crates/driver/src/boundary/liquidity/balancer/v2/mod.rs

```diff
@@ -75,19 +76,22 @@ fn to_interaction(
 }
 
 pub fn collector(
+    graph_api_base_url: &Url,
```

Comment:
this field should probably, like other things that configure the different liquidity sources, be part of the `config` struct (otherwise this method will soon have 100 arguments)

Reply:
done. PTAL